sample/config.ini

```diff
@@ -2,6 +2,10 @@
 project_name=NcEngine Samples
 log_file_path=Sample.log
 log_max_file_size=1000000
+[engine_settings]
```

Comment:
Is this something we can easily obfuscate if we were to make, for example, a P2P networked game? We would not want users to be able to modify their time_step like this in that setting.

Reply:
Ignore, we discussed this

Reply:
We addressed my other concerns in person. Looks good!